descriptors/ModuleDescriptor-template.json

```
@@ -106,6 +106,24 @@
         }
       ]
     },
+    {
+      "id": "change-due-date",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/batch/change-due-date",
```

This operation is specific to loans, I suggest that be reflected in the name endpoint e.g.
```suggestion
          "pathPattern": "/circulation/loans/batch-change-due-dates",
```

---

Renamed.